docs/usage.md

```diff
@@ -1,76 +1,80 @@
 # Usage
 
-Examples:
-
-* For [Go modules](#example-go-modules) (most complete explanation)
-* For [pip](#example-pip)
-
-General process:
-
-1. [pre-fetch dependencies](#pre-fetch-dependencies)
-2. [generate environment variables](#generate-environment-variables)
-3. [inject project files](#inject-project-files)
-4. set the environment variables ([Containerfile example](#write-the-dockerfile-or-containerfile))
-5. run the build ([container build example](#build-the-container))
-
-## Example: Go modules
-
-Let's show Cachi2 usage by building the glorious [fzf](https://github.com/junegunn/fzf) CLI tool hermetically. To follow
-along, clone the repository to your local disk.
-
-```shell
-git clone https://github.com/junegunn/fzf --branch=0.34.0
-```
-
-The best way to run `cachi2` is via the [container image](../README.md#container-image).
+This document is split into two general sections. The first goes through the general process of prefetching
```

Comment:
We are using both `pre-fetch`  and `prefetch` here. It seems that `prefetch` is the correct or most used form (or at least that's what Google thinks about it). 

Either way, I think we should keep a single form consistently.

Reply:
Unifying on pre-fetch as that was the prior art.